src/Matches.elm

```diff
@@ -38,16 +38,20 @@ getMatch match =
             getAt (n - 1) matches
 
 
-stationIndex : Maybe Int -> Maybe Int -> String
-stationIndex team match =
-    -- team2 -> index = 2
+getTeam : Int -> Int -> String
+getTeam team index =
+    "not yet"
+
+
+stationName : Maybe Int -> Maybe Int -> String
+stationName team match =
```

Comment:
[This](https://github.com/orbit1690/Scouting-2019/pull/2#discussion_r353380059) is still relevant:
> Notice how these two withDefaults are stuck there for no reason?
If we use a better design - one that makes sure we have valid data on the type level, we won't need them here 🙂.

`Maybe a`s are usually useful for validations - for functions of the form `a -> Maybe a` - where you can chain validations one after the other in succession (using [Maybe.andThen](https://package.elm-lang.org/packages/elm/core/latest/Maybe#andThen)).
My point is that your function shouldn't be bothered with invalid `team`s or `match`es - you can handle them gracefully outside. In this case, [Maybe.map2](https://package.elm-lang.org/packages/elm/core/latest/Maybe#map2) comes handy:
```elm
stationName : Int -> Int -> String
stationName team match =
    -- your implementation here
```
and then
```elm
Maybe.map2 stationName (String.toInt model.team) (String.toInt model.match)
```
results a `Nothing` if any of them is a `Nothing`.